UnitTests/GitUI.Tests/Script/ScriptOptionsParserTests.cs

```diff
@@ -152,15 +152,38 @@ public void Parse_should_return_unmodified_arguments_if_no_options_matched()
         [Test]
         public void Parse_should_parse_c_arguments()
         {
+            const string Subject = "line1";
             var revision = new GitRevision(ObjectId.IndexId)
             {
-                Subject = "line1"
+                Subject = Subject,
+                Body = $"{Subject}\n\nline3"
             };
-            _module.GetRevision(shortFormat: true, loadRefs: true).Returns(x => revision);
+            _module.GetRevision(shortFormat: false, loadRefs: true).Returns(x => revision);
+
+            string expectedMessage = $"{Subject}\\n\\nline3";
+
+            var result = ScriptOptionsParser.Parse("echo {{cSubject}} {{cMessage}}", module: _module, owner: null, scriptHostControl: null);
+
+            result.arguments.Should().Be($"echo \"{revision.Subject}\" \"{expectedMessage}\"");
```

Comment:
```suggestion
            result.arguments.Should().Be($"echo \"{revision.Subject}\" \"{revision.Body}\"");
```
...and remove `expectedMessage`.

Reply:
This cannot work with `revision.Body` because it contains linefeeds but in `expectedMessage` they are escaped as two characters `\n`.